src/Compilers/CSharp/Test/Symbol/Symbols/PartialPropertiesTests.cs

```diff
@@ -680,6 +703,127 @@ partial class C
                 );
         }
 
+        /// <summary>Based on <see cref="ExtendedPartialMethodsTests.Extern_Symbols" /> as a starting point.</summary>
+        [Fact]
+        public void Extern_03()
+        {
+            var source = """
+                using System.Runtime.InteropServices;
+
+                public partial class C
+                {
+                    public static partial int P { get; set; }
+                    public static extern partial int P
+                    {
+                        [DllImport("something.dll")]
+                        get;
+
+                        [DllImport("something.dll")]
+                        set;
+                    }
+                }
+                """;
+            var verifier = CompileAndVerify(
+                [source, IsExternalInitTypeDefinition],
+                symbolValidator: module => verify(module, isSource: false),
+                sourceSymbolValidator: module => verify(module, isSource: true));
+            verifier.VerifyDiagnostics();
+
+            void verify(ModuleSymbol module, bool isSource)
+            {
+                var prop = module.GlobalNamespace.GetMember<PropertySymbol>("C.P");
+                Assert.True(prop.GetPublicSymbol().IsExtern);
+                verifyAccessor(prop.GetMethod!);
+                verifyAccessor(prop.SetMethod!);
+
+                if (isSource)
```

Comment:
nit: We commonly do `bool inSource = module is SourceModuleSymbol;` to avoid the extra parameter, thus allowing for `symbolValidator : verify, sourceSymbolValidator: verify`

Reply:
I'll keep this in mind for future tests